SECP-service/src/main/java/com/visucius/secp/DTO/UserRegistrationRequest.java

```diff
@@ -24,4 +24,13 @@ public UserRegistrationRequest()
     {
 
     }
+
+    public UserRegistrationRequest(String firstName, String lastName, String userName ,String email, String password)
```

Comment:
I thought we agreed there is no need for DTO classes. Please just use the model directly.

Reply:
The reason i kept it is because I thought we can reuse this for admin registration. And from what i seen from read from this, https://softwareengineering.stackexchange.com/questions/171457/what-is-the-point-of-using-dto-data-transfer-objects , it seems to be good practice to use DTO. But I am not sure what do you think?

Reply:
Sure. You can keep it if you like it.